user_manual/files/federated_cloud_sharing.rst

```diff
@@ -14,8 +14,7 @@ by default. Follow these steps to create a new share with other Nextcloud or own
 
 1. Go to your ``Files`` page and click the **Share** icon on the file or directory 
 you want to share. In the sidebar enter the username and URL of the remote user
-in this form: ``<username>@<oc-server-url>``. In this example, that is
-``layla@remote-server/nextcloud``. The form automatically echoes the address 
+in this form: ``<username>@<oc-server-url>``. The form automatically echoes the address
```

Comment:
```suggestion
in this form: ``<username>@<oc-server-url>``. The form automatically confirms the address
```

"echo" is kind of techy. I'd substitute that with a more human word.

Reply:
For me _confirms_ is less clear in this context than _echoes_... but I guess I am kind of techy :-P So done :+1: